homeassistant/components/config/config_entries.py

```diff
@@ -486,10 +487,10 @@ async def async_matching_config_entries(
     entries = [
         entry
         for entry in entries
-        if (type_filter != "helper" and entry.domain not in integrations)
+        if entry.domain not in integrations
```

Comment:
I was not entirely sure why the "helper" check was here. I've not come across a test case, or reason with it was filtered in those cases.

Reply:
Figure out why, going to adjust.